src/bitmap/ops.rs

```diff
@@ -571,7 +571,12 @@ fn try_simple_multi_op_owned<E>(
     let mut iter = bitmaps.into_iter();
     match iter.next().transpose()? {
         Some(mut lhs) => {
-            iter.try_for_each(|rhs| rhs.map(|rhs| op(&mut lhs, rhs)))?;
+            for rhs in iter {
+                if lhs.is_empty() {
+                    return Ok(lhs);
+                }
+                op(&mut lhs, rhs?);
+            }
```

Comment:
I am not sure it is the right way to do so. We don't know the `op` function at this point and can't stop the loop here without knowing its real intent.

Reply:
We know it's either `sub` or `and` and it works for both though.
But maybe the name of the function should be more explicit